src/Abp/EntityHistory/EntityPropertyChange.cs

```diff
@@ -1,6 +1,9 @@
-﻿using Abp.Domain.Entities;
+﻿using System.Reflection;
```

Comment:
I recommend against `using System.Reflection` in entities.

Reply:
Moved truncation back to `EntityHistoryHelper` for now